canarytokens/windows_fake_fs.py

```diff
@@ -0,0 +1,179 @@
+
+
+DUMMY_FOLDER_STRUCTURE = [
+    {
+        "name": "Projects",
+        "type": "folder",
+        "children": [
+            {
+                "name": "Project A",
+                "type": "folder",
+                "children": [
+                    {"name": "Doc1.pdf", "type": "pdf"},
+                    {"name": "Doc2.docx", "type": "docx"},
+                    {"name": "Doc3.xls", "type": "xls"},
+                    {"name": "Doc4.doc", "type": "doc"},
+                ],
+            },
+            {
+                "name": "Project B",
+                "type": "folder",
+                "children": [
+                    {"name": "Doc A.pdf", "type": "pdf"},
+                    {"name": "Doc B.docx", "type": "docx"},
+                    {"name": "Doc C.xls", "type": "xls"},
+                    {"name": "Doc D.doc", "type": "doc"},
+                ],
+            },
+        ],
+    },
+    {
+        "name": "Testing",
+        "type": "folder",
+        "children": [
+            {"name": "Doc1.pdf", "type": "pdf"},
+            {"name": "Doc2.xlsx", "type": "xls"},
+            {"name": "Doc3.doc", "type": "doc"},
+        ],
+    },
+    {
+        "name": "Deployment",
+        "type": "folder",
+        "children": [
+            {"name": "doc a.pdf", "type": "pdf"},
+            {"name": "doc b.docx", "type": "docx"},
+            {"name": "doc c.xls", "type": "xls"},
+            {"name": "doc d.doc", "type": "doc"},
+        ],
+    },
+    {
+        "name": "Invoicing",
+        "type": "folder",
+        "children": [
+            {"name": "Invoice A.pdf", "type": "pdf"},
+            {"name": "Invoice B.xlsx", "type": "xls"},
+            {"name": "Invoice C.docx", "type": "docx"},
+            {"name": "Invoice D.doc", "type": "doc"},
+        ],
+    },
+]
+
+
+FOLDER_MAP = {
+    "personal_finances": PERSONAL_FINANCES_STRUCTURE,
+    "home_network": HOME_NETWORK_STRUCTURE,
+    "personal_correspondence": PERSONAL_CORRESPONDENCE_STRUCTURE,
+    "photo_archive": PHOTO_ARCHIVE_STRUCTURE,
+    "defense": DEFENSE_STRUCTURE,
+    "med_tech": MED_TECH_STRUCTURE,
+    "network_admin": NETWORK_ADMIN_STRUCTURE,
+    "security_admin": SECURITY_ADMIN_STRUCTURE,
+    "testing": DUMMY_FOLDER_STRUCTURE,
+}
+
+
+def _gen_ts() -> str:
+    """
+    Generates a random timestamp in the near past
+    """
+    now = floor(datetime.now().timestamp())
+    random_hours = random.randint(1, 10000)
+    return str(now - (random_hours * 60 * 60))
+
+
+def _new_item(path: str, is_folder: bool, size: int = 0) -> dict:
+    isdir = "false"
+    if is_folder:
+        isdir = "true"
+    return {"path": path, "isdir": isdir, "size": size, "timestamp": _gen_ts()}
+
+
+def _process_item(item: dict, path: str) -> str:
+    out = []
+    if item["type"] == "folder":
+        out.append(_new_item(path=path + "\\" + item["name"], is_folder=True))
+        for c in item["children"]:
+            out += _process_item(c, path + "\\" + item["name"])
```

Comment:
```suggestion
        out.extend(_process_item(c, f"{path}\\{item['name']}" for c in item["children"])
```
More concise, but feel free to ignore

Reply:
I cleaned it up using fstring but a single line for won't work here